doc/source/whatsnew/v0.20.0.txt

```diff
@@ -521,6 +521,7 @@ Other Enhancements
 - The ``usecols`` argument in ``pd.read_csv()`` now accepts a callable function as a value  (:issue:`14154`)
 - The ``skiprows`` argument in ``pd.read_csv()`` now accepts a callable function as a value  (:issue:`10882`)
 - The ``nrows`` and ``chunksize`` arguments in ``pd.read_csv()`` are supported if both are passed (:issue:`6774`, :issue:`15755`)
+- ``pd.read_csv()`` now treats 'n/a' strings as missing values by default (:issue:`16078`)
```

Comment:
Thanks for the feedback, but could you clarify what exactly would you like me to change?